assets/app/view/log.rb

```diff
@@ -3,17 +3,25 @@
 module View
   class Log < Snabberb::Component
     needs :log
+    needs :follow_scroll, default: true, store: true
 
     def render
       scroll_to_bottom = lambda do |vnode|
         elm = Native(vnode)['elm']
         elm.scrollTop = elm.scrollHeight
       end
 
+      scroll_handler = lambda do
```

Comment:
lambd do |event|

Reply:
I couldn't figure out what to do with the event once I had it. It didn't respond to `inspect`or to `target`.  How does one access the event target element in Snabberb?

I tried:
```
scroll_handler = lambda do |e|
  puts e
end
```

```
scroll_handler = lambda do |e|
  elm = Native(e.target)['elm']
  puts elm.scrollTop
end
```

In general, how do you debug Opal? I tried throwing a `debugger` in there, but that barfed.

Reply:
it will be much easier if you join the slack

the e is a js event so it cannot respond to inspect
you'll have to do

`console.log(e)`

Reply:
use back ticks
https://github.com/opal/opal/blob/master/docs/compiled_ruby.md#inline-javascript